build_osx.sh

```diff
@@ -17,6 +17,7 @@ if [ -f "$cxml" ]; then
 	python3.7 src/bindings/pyrpr/src/pyrprapi.py $cxml
 	if [ -f "./bindings-ok" ]; then
 		python3.7 build.py
+		python3.9 build.py
```

Comment:
this implies that we have to have both?  Can we build just one or the other?

Reply:
Yes, we need both currently. I think whole build process isn't optimized and requires to refactor and improve. I suggest to do this as separate task in future.